configure.ac

```diff
@@ -298,7 +298,7 @@ AC_SUBST_DEFINE(HAVE_SWRESAMPLE, $libswresample_HAVE)
 # map avutil library version to ffmpeg version
 AC_MSG_CHECKING([version of ffmpeg])
 # the order is ([avutil], [avcodec], [avformat], [swscale], [swresample])
-if   FFMPEG_LIBS_COMPATIBLE([59008100], [61003100], [61001100], [8001100], [5001100]); then
+if   FFMPEG_LIBS_COMPATIBLE([59039100], [61019100], [61007100], [8003100], [5003100]); then
```

Comment:
This is wrong. You have to list the lowest supported version and optionally additionally the highest supported version. For 7.1 this line does not need to be changed because the major version of all libraries has not changed.

Reply:
Okay, that explains a lot of things - thanks 👍🏼